### PR TITLE
Mark 3.1.56 as released

### PR DESCRIPTION
**Pull request by aheejin**

### CODE REVIEW

**Review by kripken — APPROVED** [review]
lgtm (after emsdk PR lands)

**Comment by aheejin** [comment]
3.1.56 turned out to be broken. Some context is in discussions in https://github.com/emscripten-core/emsdk/pull/1353. It looks we have to do a new release that contains #21523 and #21515, which will be marked as 3.1.57. Will land this anyway given that 3.1.56 has been (sadly) effectively shipped.

**Pull request merged**
